src/utils/UndoManager.js

```diff
@@ -372,6 +377,8 @@ export class UndoManager extends Observable {
   }
 
   destroy () {
+    if (!this.shouldDestoryUndoManager) return
+
     this.trackedOrigins.delete(this)
     this.doc.off('afterTransaction', this.afterTransactionHandler)
```

Comment:
The following code should always be executed as it is specific to the binding. 

```js
    this.trackedOrigins.delete(this)
    this.doc.off('afterTransaction', this.afterTransactionHandler)
```

Reply:
Sorry, this won't work for me, because in my case, prosemirror editors are nested in the same Y.Doc with other componenets, if afterTransaction event is unregisterred, then this undoManager will stop working for other components of this Y.Doc. It seems that there only one way left for me, leave the destroy controll to me.

Reply:
For now, I must overwrite prosemirror yUndoPlugin's view function to disable destroy calls.